db/migrations/20230706150233_endpoints_evals.sql

```diff
@@ -0,0 +1,48 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE unweave.eval (
+    id text NOT NULL PRIMARY KEY,
+    exec_id text NOT NULL REFERENCES unweave.exec (id),
+    project_id text NOT NULL REFERENCES unweave.project (id),
+    http_address text NOT NULL,
+    created_at timestamp with time zone DEFAULT now() NOT NULL
+);
+
+CREATE TABLE unweave.endpoint (
+    id text NOT NULL PRIMARY KEY,
+    exec_id text NOT NULL REFERENCES unweave.exec (id),
+    project_id text NOT NULL REFERENCES unweave.project (id),
+    http_address text NOT NULL,
+    created_at timestamp with time zone DEFAULT now() NOT NULL,
+    deleted_at timestamp with time zone
+);
+
+CREATE TABLE unweave.endpoint_eval (
+    endpoint_id text NOT NULL REFERENCES unweave.endpoint (id),
+    eval_id text NOT NULL REFERENCES unweave.eval (id),
+    PRIMARY KEY (endpoint_id, eval_id)
+);
+
+CREATE TABLE unweave.endpoint_check (
+    id text NOT NULL PRIMARY KEY,
+    endpoint_id text NOT NULL REFERENCES unweave.endpoint(id),
+    project_id text NOT NULL REFERENCES unweave.project (id),
+    created_at timestamp with time zone DEFAULT now() NOT NULL
+);
+
+CREATE TABLE unweave.endpoint_check_step (
+    id text NOT NULL PRIMARY KEY,
+    check_id text NOT NULL REFERENCES unweave.endpoint_check (id),
+    eval_id text NOT NULL REFERENCES unweave.eval (id),
+    input text,
+    output text,
+    assertion text
```

Comment:
What is the assertion field again? Is this the result of the assertion or the _expected_ result?

Reply:
it's a status returned from the eval, like `success` or `failure`